functions-and-scope.js

```diff
@@ -16,6 +16,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+// Waarde checken van de array? Ik moet een for loop gebruiken die door de waardes van de grades array heen loopt en ze 1 voor 1 chckts
+// Door grades.length te gebruiken loopt de for loop net zo lang als de array
+// if statement per loop laten loggen dat er 1 voldoet aan cum laude
+
+let output = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        output += 1;
```

Comment:
Je zou dit nog iets kunnen verkorten:
```suggestion
        output++;
```